doc/develguide.rst

```diff
@@ -0,0 +1,656 @@
+Developer Guide
+===============
+
+Overview
+--------
+
+This document describes the guts of jpype.  It is intended 
+lay out the architecture of the jpype code to aid intrepid lurkers 
+to develop and debug the jpype code once I am run over by a bus.
+
+
+History
+~~~~~~~
+
+When I started work on this project it had already existed for over 10 years.
```

Comment:
you could mention who you mean by "I", e.g. your github name etc.

Reply:
Sure will do.